### PR TITLE
feat(ui): [BACK-1331] Make items with a recomendation status visually distinct

### PR DESCRIPTION
**Pull request by nina-py**
## Goal

Make the UI better! 

Removed the recommendation/corpus list item in the item card and added an overlay
with the word "Recommendation" on images for recommended items instead. _Update_: following feedback, this is now just "Rec" on a solid Pocket teal background - see "Final version" screenshot.

Moved "Topic" and "Language" properties to the bottom of the card, so that they're always
there. Moved "Syndicated", "Collection", "Time sensitive" props up. Removed dividers and secondary colour ("Syndicated", etc. were greyed out a bit) so that there is less visual clutter.

### Before:
before

### After:

recommendations-stand-out

### Final version:

final

## Reference

Tickets:

- https://getpocket.atlassian.net/browse/BACK-1331

### CODE REVIEW

File: src/curated-corpus/components/ScheduledItemCardWrapper/ScheduledItemCardWrapper.styles.tsx [hunk 1]
```diff
@@ -9,6 +9,7 @@ export const useStyles = makeStyles((theme: Theme) =>
       height: '100%',
       display: 'flex',
       flexDirection: 'column',
+      position: 'relative',
     },
```
Comment (nina-py) [inline src/curated-corpus/components/ScheduledItemCardWrapper/ScheduledItemCardWrapper.styles.tsx:13]: I'm pretty sure this is totally against all possible CSS standards and conventions but it works.

**Review by Herraj — APPROVED** [review]
Nice!

**Review by kkyeboah — DISMISSED** [review]
looks good to me

File: src/curated-corpus/components/ApprovedItemListCard/ApprovedItemListCard.tsx [hunk 2]
```diff
@@ -49,6 +51,11 @@ export const ApprovedItemListCard: React.FC<ApprovedItemListCardProps> = (
         }
         alt={item.title}
       />
+
+      {item.status === CuratedStatus.Recommendation && (
+        <div className={classes.imageOverlay}>Rec.</div>
```
Comment (kkyeboah) [inline src/curated-corpus/components/ApprovedItemListCard/ApprovedItemListCard.tsx:56]: I prefer this without the `.`

Comment (nina-py) [reply src/curated-corpus/components/ApprovedItemListCard/ApprovedItemListCard.tsx]: Updated to just "Rec"!

**Review by Herraj — APPROVED** [review]

**Pull request merged**